app/app.tsx

```diff
@@ -73,9 +73,14 @@ function App() {
     const subscription = AppState.addEventListener("change", (nextAppState) => {
       if (appState.current.match(/inactive|background/) && nextAppState === "active") {
         // App has come to the foreground!
-        // Check if the current ride id is still active
-        if (rootStore && rootStore.ride.id) {
-          rootStore.ride.isRideActive(rootStore.ride.id)
+        if (rootStore) {
+          // Check Live Activities authorization
+          rootStore.ride.checkActivityAuthorizationInfo()
```

Comment:
You are already calling this in the model `afterCreate()` method. Isn't it sufficient?

Reply:
I want to check it every time the user opens the app, to update the state when the user comes back from settings